var/spack/repos/builtin/packages/kitty/package.py

```diff
@@ -39,6 +87,12 @@ class Kitty(Package):
     depends_on("pkgconfig", type="build")
     depends_on("py-setuptools", type="build")
     depends_on("py-sphinx", type="build")
+    depends_on("py-sphinx-copybutton", type="build")
+    depends_on("py-sphinx-inline-tabs", type="build")
+    depends_on("py-sphinxext-opengraph", type="build")
+    depends_on("py-furo", type="build")
+    depends_on("py-beautifulsoup4", type="build")
+    depends_on("py-sphinx-basic-ng", type="build")
```

Comment:
Any idea where I can find this list of deps? Also, I'm surprised that these deps are only needed to install the package and not to use it.

Reply:
I added them one-by-one until the build was able to proceed. I'm pretty certain that they are only needed to generate the docs pages which are generated as part of the default build process.

Reply:
Is it possible to disable the docs build?

Reply:
From what I can tell, you would need to patch the `setup.py`. If that is the only way, I would prefer not to do that.